### ISSUE TITLE
[build] Task `watch` do not update files that depends on site.variable

### ISSUE THREAD

**Issue by exoego** [fomantic/Fomantic-UI#631]
# Bug Report

## Steps to reproduce
1. `mkdir fui-test; cd fui-test`
2. `npm install --save fomantic-ui@2.7.4`
3. Choose `Automatic install`
4. `cd semantic`
5. `gulp build`
6. `gulp watch`
7. In different terminal, update `src/site/globals/site.variables` like below

```less
// those variables are widely used in many components
@red: #ff0000;
@h1 : unit(123, rem);
@textColor: rgba(210, 200, 200, 1.0);
```
8. Wait until watch task updates the files.

## Expected result

Every components that depends on modified variables should be re-generated.
This is possible in Semantic-UI@2.4.2.

## Actual result

It seems like that only `site(.min).css` are updated when `site.variables` are modified.
Other components are not updated and use old variables.
So, manully running `gulp build` is required to updated other components .

In short, the gulp task `watch` is useless.

## Version
2.7.4
